### PR TITLE
Create ambiclimate alert

### PR DESCRIPTION
**Pull request by codyc1515**
Sourced from https://www.ambiclimate.com/blogs/news/an-emotional-farewell-important-update-on-termination-of-ambi-climate-services

### CODE REVIEW

**Comment by codyc1515** [comment]
No one is going to review this?

File: alerts/ambiclimate.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+## Summary
+
+All Ambi Climate services will be terminated, effective March 31, 2024, as Ambi Labs winds down business operations. You will be able to continue to use the Ambi Climate service until the end of March 2024.
+
+From April 1, 2024, Ambi Climate devices will no longer be able to connect to the Ambi Climate servers, and you will no longer be able to sign into your account.
+
+As Ambiclimate devices require a connection to the manufacturers cloud server, they will cease to function from April 1 2024 onwards. We recommend removing the integration from your Home Assistant installation once this happens.
+
+Ambiclimate is allowing customers to export their private data, as well as offering a voluntary compensation policy for some customers. View the Ambiclimate website for more details
```
Comment (MartinHjelmare) [inline alerts/ambiclimate.md:16]:
```suggestion
Ambiclimate is allowing customers to export their private data, as well as offering a voluntary compensation policy for some customers. View the Ambiclimate website for more details.
```

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Comment by MartinHjelmare** [comment]
We're also deprecating the core integration.

**Pull request merged**